examples/telegram-bot/docker-compose.yml

```
@@ -2,7 +2,7 @@ version: "3"
 
 services:
   api:
-    image: quay.io/go-skynet/local-ai:v1.18.0-ffmpeg
+    image: quay.io/go-skynet/local-ai:v2.17.1-ffmpeg
```

oops, good catch - since we are here maybe worth switching to use `latest` so we don't have to keep that up-to-date?

```suggestion
    image: quay.io/go-skynet/local-ai:latest-ffmpeg
```

---

Was also thinking about that. Let me test with the latest tomorrow and then redo the PR, and also sign it.

---

As there is no quay.io/go-skynet/local-ai:latest-ffmpeg - I tested with quay.io/go-skynet/local-ai:latest-cpu
```
 ✘ api Error manifest for quay.io/go-skynet/local-ai:latest-ffmpeg not found: manifest unknown: manifest unknown                                                                                                                                          2.0s 
Error response from daemon: manifest for quay.io/go-skynet/local-ai:latest-ffmpeg not found: manifest unknown: manifest unknown
```
So far everything worked as before, which makes me wonder what the purpose is behind choosing ffmpeg. I fear something I'm not aware of might break. Where would I look/test to check?

---

ouch sorry, yes, there is only `:latest`, or `:latest-ffmpeg-core`